docs/guide/markdown.md

```diff
@@ -392,7 +392,7 @@ Additionally, you can define a number of lines to focus using `// [!code focus:<
 export default {
   data () {
     return {
-      msg: 'Focused!' // [!codeㅤ focus]
+      msg: 'Focused!' // [!codeㅤfocus]
```

Comment:
This is not a space either. It's U+3164. Having such characters in source code is worse than having two spaces. It is more likely to go unnoticed than double spaces!

Reply:
Yeah, I missed that too. It is there in the current docs too. I'd missed them while reviewing earlier PRs. Gonna remove all those in #1615.